topology/University of Arizona/Research Technologies/ResearchTechnologies.yaml

```
@@ -0,0 +1,25 @@
+GroupDescription: ResearchTechnologies
+GroupID: 1366
+Production: true
+Resources:
+  Gateway:
```

```suggestion
  Arizona-ResearchTechnologies-Gateway:
```
Resource names need to be globally unique so you should use a less generic name.